imgee/templates/form.html.jinja2

```diff
@@ -1,11 +1,11 @@
 {% extends "layout.html.jinja2" %}
 {% from 'baseframe/forms.html.jinja2' import renderform %}
 {% from "baseframe/components.html.jinja2" import faicon %}
-{% block title %}New image{% endblock %}
+{% block title %}{% trans %}New image{% endtrans %}{% endblock %}
 
 {% block maincontent %}
 	<div class="mui-panel">
-		<p class="mui--text-title mui--text-bold">{{ faicon(icon='upload', icon_size='subhead', baseline=true) }}&nbsp;&nbsp;Upload Image</p>
+		<p class="mui--text-title mui--text-bold">{{ faicon(icon='upload', icon_size='subhead', baseline=true) }}&nbsp;&nbsp;{% trans %}Upload Image{% endtrans %}</p>
```

Comment:
Sentence case: Upload image

@vidya-ram this template has 2x `&nbsp;` between icon and text, but the delete template above has only one. This needs revising.

Reply:
Add css class to include margin